### PR TITLE
Add github actions to implement CI, remove STDCI support

### PR DESCRIPTION
**Pull request by sjd78**
GitHub Actions to implement CI changes:
  - Two parallel sets of jobs run.  One set test then build using yarn offline / ovirt-engine-nodejs-modules.  The other set test and build using yarn online / direct download of dependencies.

  - If the test job fails, the build job will not run.

  - Only the offline built rpms are exported.

  - Refactored the build scripts to build in `$repo_root/tmp.repos`.

  - Added exclusions / ignores as appropriate to `tmp.repos/`.

  - Set `.eslintrc.js` to be the root config.  If this is not done, eslint will look in every parent for another config and cause annoying problems when the rpm is being built.

Remove STDCI support changes:
  - Remove `automation.yaml` and related `automation/*.packages`, `automation/*.repos` and `automation/*.sh` files.  Without the base yaml file, std-ci won't attempt to run anything.

  - Github actions take over PR CI testing and builds

  - copr provides builds from the master branch (https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/)

  - `automation/build.sh` is retained with specific accommodations for std-ci/jenkins package caching issues removed

### CODE REVIEW

File: .github/workflows/check.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#
+# CI for this project needs to do two things:
+#   1. Setup the environment for and run `yarn build`
```
Comment (michalskrivanek) [inline .github/workflows/check.yaml:3]: s/for// ?

File: .github/workflows/check.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+on:
+  pull_request:
+
+jobs:
+  test_el8_offline:
+    name: EL8 - test the PR (ovirt-engine-nodejs-modules build)
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
```
Comment (michalskrivanek) [inline .github/workflows/check.yaml:16]: please use quay.io/ovirt/buildcontainer:el8stream (https://github.com/oVirt/buildcontainer), it has all deps installed already

Comment (michalskrivanek) [reply .github/workflows/check.yaml:16]: and you can drop all the additional dnf installs below.

**Review by michalskrivanek — CHANGES_REQUESTED** [review]
looks good, let's just update for buildcontainer. 
adding matrix for el9 canbe done either now or later...

**Comment by sjd78** [comment]
> adding matrix for el9 canbe done either now or later...

Actually, as I found out yesterday, any project using a webpack version less than 5.61.0 cannot build on el9.  Ovirt projects are on webpack 4.  There was a change to openssl and nodejs to broke a hash function used by webpack.

The only way for our projects to build on el9 is to upgrade to webpack 5.

**Comment by sjd78** [comment]
Replaced with #24

**Pull request closed without merging**